colly.go

```diff
@@ -693,6 +693,11 @@ func (c *Collector) isDomainAllowed(domain string) bool {
 	if c.AllowedDomains == nil || len(c.AllowedDomains) == 0 {
 		return true
 	}
+	for _, d2 := range c.AllowedDomains {
+		if strings.Contains(domain, d2) {
```

Comment:
I recommend to use "HasSuffix" method instead of "Contains". Does that make sense?

Reply:
yeah. "HasSuffix" is well

Reply:
HasSuffix sounds better for me, could you change it? Also, please update the docstring with the new behavior.